docs/user-guide/projects/project-lists/README.md

```diff
@@ -176,19 +176,22 @@ You can share a project list with everyone within your OpenProject instance. Thi
 To do that click the **Share** icon and turn on the **Share with everyone at your instance** switch.
 
 ![Share a project list with everyone in OpenProject](project-lists-share-with-everyone-button.png)
-It will be added to the menu on the left under *Shared project lists*.
+The project list will move from *My project lists* section to *Shared project lists* section in the lefthand menu.
 
-![Public project lists in OpenProject]()
+![Public project lists in OpenProject](project-lists-share-with-everyone-list.png)
 
-UPDATE SCREENSHOT
+You can reverse the action by unselecting the star icon. The project list will return to *My project lists*. 
```

Comment:
@MayaBerd I think this is not correct. Unselecting the star icon will not bring the list back to "My project lists". You need to switch off the share with everyone toggle for that.

Reply:
@marcalcobe you are right! i mixed those up, will correct.